applications/ContactMechanicsApplication/ContactMechanicsApplication.py

```diff
@@ -1,6 +1,9 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
 from __future__ import print_function, absolute_import, division
 
+# Applications requiered
+from KratosPfemApplication import * 
```

Comment:
I Know I told you that was a possible solution, but I would have preferred to discuss it before merging into the master. Since its here anyway... @KratosMultiphysics/technical-committee what  do you think. Should we allow this so people does not need to import the dependent applications?

My opinion would be no, but in the particular case of PFEM I can't see any major drawback

Reply:
it works